src/router/rules/GivenUserHasStartedEditingADifferentDraftRegistration_ThenDeleteItAndReloadPage.ts

```diff
@@ -37,7 +37,7 @@ export class GivenUserHasStartedEditingADifferentDraftRegistration_ThenDeleteItA
   private theDraftRegistrationCookieIdMatchesTheQueryId() {
```

Comment:
```suggestion
  private theDraftRegistrationCookieIdDoesNotMatchTheQueryId() {
```
I hate booleanssss, but is this a better name? 😅 I might be wrong